src/bun.js/api/server.zig

```diff
@@ -6230,6 +6241,7 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
             request_value.ensureStillAlive();
             const response_value = this.config.onRequest.callWithThis(this.globalThis, this.thisObject, &args);
             defer {
+                if (!ctx.didUpgradeWebSocket()) {}
```

Comment:
did you mean to do something here?

Reply:
I did not, probably was some log that I add and removed sorry my mistake